src/main/java/at/hannibal2/skyhanni/features/misc/discordrpc/DiscordStatus.kt

```diff
@@ -308,7 +308,7 @@ enum class DiscordStatus(private val displayMessageSupplier: (() -> String?)) {
         if (!DungeonAPI.inDungeon()) {
             AutoStatus.DUNGEONS.placeholderText
         } else {
-            val boss = DungeonAPI.getCurrentBoss()
+            val boss = DungeonAPI.dungeonFloor
```

Comment:
Some code cleanup while we're at it - GitHub won't let me make a suggestion to some lines that weren't changed in this pull request, so just apply this manually:
```diff
diff --git a/src/main/java/at/hannibal2/skyhanni/features/misc/discordrpc/DiscordStatus.kt b/src/main/java/at/hannibal2/skyhanni/features/misc/discordrpc/DiscordStatus.kt
index 42ac04a6a..b82082f90 100644
--- a/src/main/java/at/hannibal2/skyhanni/features/misc/discordrpc/DiscordStatus.kt
+++ b/src/main/java/at/hannibal2/skyhanni/features/misc/discordrpc/DiscordStatus.kt
@@ -305,18 +305,14 @@ enum class DiscordStatus(private val displayMessageSupplier: (() -> String?)) {
     }),

     DUNGEONS({
-        if (!DungeonAPI.inDungeon()) {
-            AutoStatus.DUNGEONS.placeholderText
-        } else {
-            val boss = DungeonAPI.dungeonFloor
-            if (boss == null) {
-                "Unknown dungeon boss"
-            } else {
-                val floor = DungeonAPI.dungeonFloor ?: AutoStatus.DUNGEONS.placeholderText
-                val amountKills = DungeonAPI.bossStorage?.get(boss)?.addSeparators() ?: "Unknown"
+        if (DungeonAPI.inDungeon()) {
+            DungeonAPI.dungeonFloor?.let {
+                val amountKills = DungeonAPI.bossStorage?.get(it)?.addSeparators() ?: "Unknown"
                 val time = DungeonAPI.getTime()
-                "$floor Kills: $amountKills ($time)"
-            }
+                "$it Kills: $amountKills ($time)"
+            } ?: "Unknown dungeon floor"
+        } else {
+            AutoStatus.DUNGEONS.placeholderText
         }
     }),
```